### ISSUE TITLE
Missing german translation/hardcoded strings

### ISSUE THREAD

**Issue by farosch** [mRemoteNG/mRemoteNG#1237]
***See update below***

**Comment by farosch** [comment]
I'll close the current PR #1229 and create a new one for this once the RemoveMagicLib branch is merged into develop.
I'll use this issue for me to keep track of all the translating issues I notice. These are either missing/awkward german strings ore hardcoded non-translatable ones:

**Settings**
![mremoteng_2019-01-13_11-27-00](https://user-images.githubusercontent.com/22905106/51084242-b7b80f80-1726-11e9-9b1d-38f145d74806.png)
![mremoteng_2019-01-13_11-27-20](https://user-images.githubusercontent.com/22905106/51084243-b7b80f80-1726-11e9-90e4-2e25a59f3b07.png)
![mremoteng_2019-01-13_11-27-40](https://user-images.githubusercontent.com/22905106/51084244-b7b80f80-1726-11e9-96c7-a19c30cdc6da.png)
![mremoteng_2019-01-13_11-28-07](https://user-images.githubusercontent.com/22905106/51084245-b7b80f80-1726-11e9-8c26-cad8d7b16ab9.png)
![mremoteng_2019-01-13_11-35-00](https://user-images.githubusercontent.com/22905106/51084283-4fb5f900-1727-11e9-92d0-6ff025b87301.png)

**Config**
![mremoteng_2019-01-13_11-24-11](https://user-images.githubusercontent.com/22905106/51084250-c7cfef00-1726-11e9-89e3-461037782c2f.png)

**Toolstrip menus**
![2019-01-13_11-21-42](https://user-images.githubusercontent.com/22905106/51084259-e0d8a000-1726-11e9-8a84-fb2139a2e5e2.png)
![2019-01-13_11-22-20](https://user-images.githubusercontent.com/22905106/51084260-e0d8a000-1726-11e9-95f1-241b4f7d6210.png)

**Right-click menus**
![2019-01-13_11-25-39](https://user-images.githubusercontent.com/22905106/51084262-e8984480-1726-11e9-9adf-3650adfb191b.png)
![2019-01-13_11-23-07](https://user-images.githubusercontent.com/22905106/51084263-e8984480-1726-11e9-9ef0-3afa6bcd3e5d.png)

**Update-popup**
![mremoteng_2019-01-13_11-20-46](https://user-images.githubusercontent.com/22905106/51084268-fcdc4180-1726-11e9-8c7a-533c5942a029.png)
